pdf/model/colorspace.go

```diff
@@ -509,14 +509,17 @@ func (this *PdfColorspaceDeviceCMYK) ColorToRGB(color PdfColor) (PdfColor, error
 	y := cmyk.Y()
 	k := cmyk.K()
 
-	c = c*(1-k) + k
-	m = m*(1-k) + k
-	y = y*(1-k) + k
-
+	// Peter Williams. I need a cmyk->rgb conversion that gives r!=g or g!=b when c!=m or m!=y
```

Comment:
OK.  Having the dummy encodings defined allows checking the encoding, which can then be processed without encoding/decoding.  I think that is a good idea.  But I think we should return an error when attempting to Decode/Encode, e.g. ErrNotImplemented.  Then can check for that error if one wants to process image data without failing on that error.